MultisigIntegrationTests/UI Snapshot Tests/BalanceTableViewCellTests.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  BalanceTableViewCellTests.swift
+//  MultisigIntegrationTests
+//
+//  Created by Dmitry Bespalov on 09.06.21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import XCTest
+import SnapshotTesting
+@testable import Multisig
+
+class BalanceTableViewCellTests: XCTestCase {
+    func testEthValue() {
+        let vc = CellTestViewController<BalanceTableViewCell>(estimatedHeight: 60) { cell in
+            cell.setMainText("ETH")
+            cell.setDetailText("0,005")
+            cell.setSubDetailText("155,84 USD")
+            cell.setImage(UIImage(named: "ico-ether")!)
+        }
+        assertSnapshot(matching: vc, as: .image(on: .iPhoneX))
+    }
+
+    func testNotConfigured() {
+        let vc = CellTestViewController<BalanceTableViewCell>(estimatedHeight: 60)
+        assertSnapshot(matching: vc, as: .image(on: .iPhoneX))
+    }
+
+    func testEmpty() {
+        let vc = CellTestViewController<BalanceTableViewCell>(estimatedHeight: 60) { cell in
+            cell.setMainText("")
+            cell.setDetailText("")
+            cell.setSubDetailText("")
+            cell.setImage(UIImage())
+        }
+        assertSnapshot(matching: vc, as: .image(on: .iPhoneX))
+    }
+
+    func testLongStrings() {
+        let vc = CellTestViewController<BalanceTableViewCell>(estimatedHeight: 60) { cell in
+            cell.setMainText(String(repeating: "ETH repeating ", count: 15))
```

Comment:
this needs to be fixed actually, because only amounts are shown

Reply:
so what is wrong here?